### PR TITLE
Make Flutter wallet CI work 

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**

### CODE REVIEW

**Comment by faddat** [comment]
OK, this is good.  I noticed those coloredbox issues, too. 

Do you want to merge this one as rogress toward the goal?

**Review by faddat — APPROVED** [review]

File: wallet/test/mocks/mock_transaction_gateway.dart [hunk 1]
```diff
@@ -22,6 +22,7 @@ import 'package:transaction_signing_gateway/transaction_signing_gateway.dart'
 // ignore_for_file: invalid_use_of_visible_for_testing_member
 // ignore_for_file: prefer_const_constructors
 // ignore_for_file: unnecessary_parenthesis
+// ignore_for_file: camel_case_types
```
Comment (faddat) [inline wallet/test/mocks/mock_transaction_gateway.dart:25]: These are the files where fixes failed, because of the ignite codegen issues?

**Comment by faddat** [comment]
I think I have a way to get some packages unstuck.  Jawad if you don't like the work, please feel free to revert

File: wallet/test/mocks/mock_transaction_gateway.dart (hunk 1 above)
Comment (kjawadDeveloper2) [reply wallet/test/mocks/mock_transaction_gateway.dart:25]: This is as you see a mock file . That mockito packages uses to create auto generate mocks.

File: easel/lib/main.dart [hunk 2]
```diff
@@ -42,7 +42,7 @@ Future<void> main() async {
     isTablet = MediaQueryData.fromWindow(WidgetsBinding.instance.window)
             .size
             .shortestSide >=
-        TABLET_MIN_WIDTH;
+        tableMinWidth;
```
Comment (mustafapylons) [inline easel/lib/main.dart:45]: fix typo "tabletMinWidth" instead of "tableMinWidth"

File: easel/lib/utils/constants.dart [hunk 3]
```diff
@@ -101,7 +101,7 @@ const kFileSizeLimitInGB = 32;
 const kFileSizeLimitForAudiVideoInGB = 0.2;
 const kMaxPriceLength = 14;
 const kSecInMillis = 1000;
-const double TABLET_MIN_WIDTH = 600;
+const double tableMinWidth = 600;
```
Comment (mustafapylons) [inline easel/lib/utils/constants.dart:104]: same here "tabletMinWidth"

**Review by mustafapylons — COMMENTED** [review]
Few issues, otherwise code is good.

File: easel/lib/utils/constants.dart (hunk 3 above)
Comment (kjawadDeveloper2) [reply easel/lib/utils/constants.dart]: Done

File: easel/lib/main.dart (hunk 2 above)
Comment (kjawadDeveloper2) [reply easel/lib/main.dart]: Done

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

File: easel/lib/models/nft.dart [hunk 4]
```diff
@@ -9,6 +9,8 @@ import 'package:pylons_sdk/pylons_sdk.dart';
 
 import '../utils/enums.dart';
 
+//ignore_for_file:  must_be_immutable
```
[the remark this answers is not in the document]
Comment (faddat) [reply easel/lib/models/nft.dart:12]: you saw the same that I did.

Recommend that we keep as is for now, AND I'll open a PR that does not have these, which will be for deeper digging.

So much of the issue here is ignite cli related

**Pull request merged**